enos/ci/service-user-iam/service-quotas.tf

```diff
@@ -0,0 +1,31 @@
+locals {
+  subnets_per_vps_quota = "L-F678F1CE"
```

Comment:
Can we leave a comment about what this is and why it's static?

Reply:
will do